core/src/apps/benfen/sign_tx.py

```
@@ -0,0 +1,235 @@
+
+from apps.common import paths, seed
+from apps.common.keychain import Keychain, auto_keychain
+
+from . import ICON, PRIMARY_COLOR
+from .helper import INTENT_BYTES, benfen_address_from_pubkey, try_convert_to_bfc_address
+from .layout import require_confirm_fee, require_show_overview
+from .tx_parser import TransactionParser
+
+
+async def process_transaction(
+    ctx, address: str, tx_bytes: bytes, coin_type: bytes
```

Ruff (0.8.2)

Missing type annotation for function argument `ctx`

(ANN001)